common/src/main/java/com/wynntils/models/profession/objects/ProfessionProgress.java

```diff
@@ -0,0 +1,9 @@
+/*
```

Comment:
For other models, I used the package name `type` for this kind of simple record/enum type classes.